deploy-web/src/components/liquidity-modal/index.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useCallback, useMemo } from "react";
+import { LiquidityModal as LeapLiquidityModal, Tabs, TxnSummary, defaultZIndices, defaultBlurs } from "@leapwallet/elements";
+import type { ThemeDefinition, WalletClient, TabsConfig, AssetSelector, AllowedDestinationChainConfig } from "@leapwallet/elements";
+import type { StdSignDoc } from "@cosmjs/amino";
+import type { SignDoc } from "cosmjs-types/cosmos/tx/v1beta1/tx";
+import { useWallet } from "@src/context/WalletProvider";
+import Button from "@mui/material/Button";
+
+const theme: ThemeDefinition = {
+  colors: {
+    primary: "#E85A39",
+    primaryButton: "#E85A39",
+    border: "#282828",
+    stepBorder: "#383838",
+    backgroundPrimary: "#212121",
+    backgroundSecondary: "#141414",
+    text: "#ffffff",
+    textSecondary: "#9E9E9E",
+    primaryButtonText: "#ffffff",
+    alpha: "#D6D6D6",
+    gray: "#9E9E9E",
+    error: "#ff6961",
+    errorBackground: "#272727",
+    success: "#29A874",
+    successBackground: "#0D3525",
+    warning: "#ffa726"
+  },
+  zIndices: defaultZIndices,
```

Comment:
The website header has a zIndex of `1100` and goes over the modal if it uses the `defaultZIndices` (`1000`).
```suggestion
  zIndices: { modalOverlay: "1200" },
```

Reply:
Thanks for the note!